pure3.md

```diff
@@ -0,0 +1,169 @@
+# PURE-3
+
+* PURE: 3
+* Title: Scudo: Adding Supply Chain Security Capabilities to Uptane
+* Version: 1
+* Last-Modified: 2022-03-30
+* Authors: Marina Moore <>, Aditya Sirish <aditya.sirish@nyu.edu>, Lois Anne Delong <>
+* Status: Draft
+* Content-Type: <text/markdown>
+* Created: 2022-03-30
+
+## Abstract
+
+## Motivation
+
+Software supply chain attacks are on the rise with the
+[2021 State of the Software Supply Chain report](https://www.sonatype.com/hubfs/SSSC-Report-2021_0913_PM_2.pdf?hsLang=en-us),
+noting a 650% increase in 2021. But, it was the
+[2020 Sunburst attack](https://www.cynet.com/attack-techniques-hands-on/sunburst-backdoor-c2-communication-protocol/) that really made both
+the government and corporate CISOs take action.
+Affecting more than 100 companies and 9 government agencies, Sunburst malware was uploaded via
+a routine update from Orion software produced by SolarWinds. The attack taught both
+public and private concerns some hard lessons, such as:
+
+* The very thing designed to protect the security of software-patches and updates that fix routine vulnerabilities-can also be used to deliver malware that can damage or corrupt.
+* As software products contain third-party components, program developers are
+  not in complete control of the contents of their products.
+* Malware can linger undetected for extended periods of time, allowing it to
+  infect every system that touches it. The Sunburst malicious code was part
+  of an update released by SolarWInds in the spring of 2020, yet it was not
+  reported until December of that year.
+
+Automotive software supply chains are particularly tempting targets for attackers. This is
+because automotive software runs on a large number of vehicles and can be difficult
+> [Something is off in the second part of this sentence]
```

Comment:
```
Automotive software supply chains are particularly tempting targets for
attackers. This is because automotive software runs on a large number of
vehicles and updating all of them to recover quickly from an attack is a
difficult proposition. As such, strategies that can secure automotive software
supply chains are vital.
```

Does this read better?

Reply:
Much better.